src/main/index.ts

```diff
@@ -128,9 +129,67 @@ log.info("Application launched");
 // Enforce sandbox on all renderers
 app.enableSandbox();
 
-// Disabling the application menu improves performance, but we're disabling it to patch out default hotkeys for now
-// TODO: Come back to this at a future point in time to determine the necessary need for an application menu
-Menu.setApplicationMenu(null);
+const application: MenuItemConstructorOptions = {
+  label: "Application",
+  submenu: [
+    {
+      label: "About Application",
+      role: "about"
+    },
+    {
+      type: "separator"
+    },
+    {
+      label: "Quit",
+      accelerator: "Command+Q",
+      click: () => {
+        app.quit();
+      }
+    }
+  ]
+};
```

Comment:
Could this just be done with just `{ role: "appMenu" }`? I'm not sure what this provides on mac and whether it may be more useful or have too much that the application doesn't need or some functionality of it end up broken.

Side note `appMenu` is ONLY available on macOS so that needs to be platform guarded.